becquerel/core/rebin.py

```diff
@@ -394,63 +301,65 @@ def rebin(in_spectra, in_edges, out_edges, method="interpolation",
         The rebinned spectrum/a
     """
     method = method.lower()
+    if not hasattr(in_spectra, "shape"):
+        assert xarray is False, "Use xarray as input not list"
```

Comment:
Maybe I missed this ... where is `xarray` defined? Is this left over?

Reply:
Yeah left over. Thanks for looking through this so thoroughly. Removed it now.